torch/nn/functional.py

```diff
@@ -4680,6 +4680,8 @@ def triplet_margin_loss(
         reduction_enum = _Reduction.legacy_get_enum(size_average, reduce)
     else:
         reduction_enum = _Reduction.get_enum(reduction)
+    if margin <= 0:
+        raise ValueError("margin must be greater than 0")
```

Comment:
nit:
```suggestion
        raise ValueError(f"margin must be greater than 0, got {margin}")
```

Reply:
Thank you! I also changed that in triplet_margin_with_distance_loss and updated the test.